api/handlers.go

```diff
@@ -476,7 +477,7 @@ func (api *API) userPUT(w http.ResponseWriter, req *http.Request, _ httprouter.P
 		api.WriteError(w, err, http.StatusBadRequest)
 		return
 	}
-	if payload.Email == "" && payload.StripeID == "" {
+	if payload.Email == "" && payload.StripeID == "" && payload.Password == "" {
```

Comment:
Can't you compare `payload == (userUpdateData{})` ? Seems like there are no reference types there which should make that comparison possible.

Now that I'm thinking about it doesn't it feel like we should return an error here to avoid dev mistakes? Why would you ever want this to be a no-op as a client?

Reply:
Both good points. Yes, an empty update seems like a dev mistake.